src/pages/Discover/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import Card from '../../components/Card';
+import './styles.css';
+
+export default function DiscoverPage() {
+  const [places, setPlaces] = useState([]);
+
+  useEffect(() => {
+    async function getPlaces() {
+      const url =
+        'https://raw.githubusercontent.com/manypossibles/designops/master/assets/data/en/data.json';
+      const response = await fetch(url);
+      const data = await response.json();
```

Comment:
Make a folder called `services` and make a file called `VisitAarhusService.js` where you can export this call as a function. In the function you can use the JS `map` method to read the data in the format that we want it so you can return it in the correct format